syne_tune/config_space.py

```diff
@@ -32,11 +32,11 @@ class Domain:
     """Base class to specify a type and valid range to sample parameters from.
 
     This base class is implemented by parameter spaces, like float ranges
-    (`Float`), integer ranges (`Integer`), or categorical variables
```

Comment:
Mainly doing docs refactoring here, but also moved "legacy" from Ray Tune to the end, so the generated docs are easier to read.